tests/conftest.py

```diff
@@ -199,6 +209,23 @@ def _cleanup(ec2, elb, resource_name_prefix: str):
         elb.delete_load_balancer(LoadBalancerName=lb_name)
 
 
+def _cleanup(ec2, elb, resource_name_prefix: str):
+    functions = [
+        (cleanup_instances, (ec2, resource_name_prefix)),
+        (cleanup_volumes, (ec2, resource_name_prefix)),
+        (cleanup_igw, (ec2, resource_name_prefix)),
+        (cleanup_vpc, (ec2, resource_name_prefix)),
+        (cleanup_ssh, (ec2, resource_name_prefix)),
+        (cleanup_elb, (elb, resource_name_prefix)),
```

Comment:
`functools.partial` would fit nicely here but I'm fine with this approach as well.

Reply:
I also needed the method name, so I reverted the partial approach.